### PR TITLE
create new function for activation and claiming 3rd party reward

### PR DESCRIPTION
**Pull request by AlexChetverov**

### CODE REVIEW

File: cliTool/index.ts [hunk 1]
```diff
@@ -585,6 +587,23 @@ async function app(): Promise<any> {
...
+        await inquirer
+          .prompt([
+            {
+              type: "input",
+              name: "user",
+              message: "default //Charlie",
+            },
+          ])
+          .then(async (answers: { user: string }) => {
+            activateAndClaim3rdPartyRewardsForUser(answers.user);
```
Comment (goncer) [inline cliTool/index.ts:604]: plz default it as charlie as you mentioned above,

File: utils/setupsOnTheGo.ts [hunk 2]
```diff
@@ -1312,3 +1314,87 @@ export async function replaceByStateCall(
...
+    ),
+  );
+  const userRewardsTokenBalance = (
+    await mangata.query.getTokenBalance(
+      newToken.toString(),
+      testUser.keyRingPair.address,
+    )
+  ).free;
+
+  return {
```
Comment (goncer) [inline utils/setupsOnTheGo.ts:1393]: you do not need to return anything, plz remove it and print some relevant info like: 
testLog.getLog().info( `Setup rewards for token-ID {tokenId} and user {userAddress} , for until session: {sessionNumber}` )

**Review by goncer — CHANGES_REQUESTED** [review]

File: cliTool/index.ts (hunk 1 above)
Comment (AlexChetverov) [reply cliTool/index.ts:604]: Fixed

File: utils/setupsOnTheGo.ts (hunk 2 above)
Comment (AlexChetverov) [reply utils/setupsOnTheGo.ts]: Done

**Review by goncer — APPROVED** [review]

**Pull request merged**